Update chapter.md (#53)
mautic/mautic-community-handbook: pages/09.education-team/chapter.md

> To improve the resources available for new and established users to learn how to use Mautic

### General Tasks

>   * Help marketers to contribute their existing content to the Mautic knowledge base

Reviewer: ```suggestion
  * Help marketers to contribute their existing content to the [Mautic Knowledgebase](https://kb.mautic.org)
```

So far we have used all one word and capitalised - not sure if that is correct, maybe we should use two separate words! :o

>   * Update the documentation to be up to date with new releases

Reviewer: ```suggestion
  * Ensure that the documentation is up to date when new releases are made
```

>   * Improve users experience by making the documentation high quality and comprehensive

Reviewer: ```suggestion
  * Improve the experience of Mautic users by ensuring that the documentation is of high quality and comprehensive
```

>   * Moderate forums and provide contents to FAQs

Reviewer: ```suggestion
  * Moderate the [Mautic Community Forums](https://forum.mautic.org) and provide content to answer FAQs
```
